specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json

```diff
@@ -5417,16 +5421,7 @@
         "protectionState": {
           "description": "Protected, ProtectionStopped, IRPending or ProtectionError",
           "type": "string"
-        },
-        "isScheduledForDeferredDelete": {
```

Comment:
this is a breaking change - removing existing properties.

Reply:
The field is moved to base class ProtectedItem and thus not a breaking change